Resources/translations/sub_items.en.xliff

```diff
@@ -86,6 +81,71 @@
+      </trans-unit>
+      <trans-unit id="9157594a9c2f44581165ee5e73c89bc5985a7fc9" resname="bulk_hide.success.message">
+        <source>The selected location(s) have been hidden.</source>
+        <target state="new">The selected location(s) have been hidden.</target>
+        <note>key: bulk_hide.success.message</note>
+      </trans-unit>
+      <trans-unit id="16c240de3cfc07b8e56d008357706abefad3e3ae" resname="bulk_unhide.popup.confirm">
+        <source>Reveal</source>
+        <target state="new">Reveal</target>
+        <note>key: bulk_unhide.popup.confirm</note>
+      </trans-unit>
+      <trans-unit id="99b9c21ff0ed8b33521c0b8c07410f1f10e5f879" resname="bulk_unhide.popup.message">
+        <source>Are you sure you want to reveal the selected location(s)?</source>
+        <target state="new">Are you sure you want to reveal the selected location(s)?</target>
+        <note>key: bulk_unhide.popup.message</note>
+      </trans-unit>
+      <trans-unit id="02b906429a29023075754b4dd3bf437b80a25ce6" resname="bulk_unhide.error.modal.table_title">
+        <source>Locations cannot be revealed (%itemsCount%)</source>
+        <target state="new">Locations cannot be revealed (%itemsCount%)</target>
+        <note>key: bulk_unhide.error.modal.table_title</note>
+      </trans-unit>
+      <trans-unit id="1208ce7816f3fa2b489c6946068c415949cf4c64" resname="bulk_unhide.error.message">
+        <source>%failedCount% of the %totalCount% selected location(s) could not be revealed because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.</source>
+        <target state="new">%failedCount% of the %totalCount% selected location(s) could not be revealed because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.</target>
+        <note>key: bulk_unhide.error.message</note>
+      </trans-unit>
+      <trans-unit id="edb883dfd8dc46cc6d99b10e9e6fd7bc708f72db" resname="bulk_unhide.success.message">
+        <source>The selected location(s) have been revealed.</source>
+        <target state="new">The selected location(s) have been revealed.</target>
+        <note>key: bulk_unhide.success.message</note>
+      </trans-unit>
+      <trans-unit id="d90e512d5cfd6d471246fad1cea365751b1cace5" resname="bulk_add_location.error.modal.table_title">
+        <source>Locations cannot be added (%itemsCount%)</source>
+        <target state="new">Locations cannot be added (%itemsCount%)</target>
+        <note>key: bulk_add_location.error.modal.table_title</note>
+      </trans-unit>
+      <trans-unit id="ba6155333e2bbe9f762d1e1462a9c3075ce38a75" resname="bulk_add_location.error.message">
+        <source>%failedCount% of the %totalCount% selected locations(s) could not be added because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.</source>
```

Comment:
```suggestion
        <source>%failedCount% of the %totalCount% selected Locations(s) could not be added because you do not have proper user permissions. {{ moreInformationLink }} Contact your Administrator to obtain permissions.</source>
```